src/tweetnacl.c

```diff
@@ -1,3 +1,16 @@
+/* Since TweetNacl doesn't come with a `randombytes` implementation,
+we're using the one from PHP.*/
+#include "php_snuffleupagus.h"
+#include "ext/standard/php_random.h"
+
+ZEND_DECLARE_MODULE_GLOBALS(snuffleupagus)
+
+void randombytes(unsigned char *x, unsigned long long xlen) {
+  php_random_bytes(x, xlen, 1);
```

Comment:
beware, php_random_byte receives `void *` and `size_t` types, shall we stick to the prototype to be sure there is not jokes ?

Reply:
If you are all good on the `size_t`, it's ok for me :)